chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+file_1=$1
+file_2=$2
+
+for FILE in $file_1, $file_2; do
+	if [ -f $FILE ]
```

Comment:
Always quote strings containing variables.
```suggestion
	if [ -f "$FILE" ]
```

Reply:
done

Reply:
No it's not 🥲

Reply:
right now?